### ISSUE TITLE
Wrong position in RecyclerView

### ISSUE THREAD

**Issue by zenkhas** [skydoves/PowerMenu#5]
![screenshot_1521524357](https://user-images.githubusercontent.com/15236686/37638302-f8a0e10a-2c2d-11e8-83fc-0d03e49762b6.png)
 
The menu is attached to the "Productive" TextView, it looks fine on other list items, but when clicked on the last shown item in list, it ties to expand into the space remaining below the item instead of top.

> contextMenu.showAsDropDown(view, 0, 0);

**Comment by skydoves** [comment]
In my case, any problems cause not yet in in RecyclerView.
I tested on __GithubFollows__ repository's __issue_powermenu__ branch.

https://github.com/skydoves/GithubFollows/tree/feature/issue_powermenu

checkout please in __MainActivity__'s testMenu.
Thank you for your issue!

**Comment by zenkhas** [comment]
![screenshot_1521530481](https://user-images.githubusercontent.com/15236686/37640860-3e1a1ad4-2c39-11e8-9fb2-6635eca9bff1.png)
It has the same problem too, if you click on the last "visible" item of the RecyclerView.

Can we make it ignore the Drop-Down behaviour when there is no space left and open up in upwards directions instead?

**Comment by skydoves** [comment]
it fixed at v2.0.2.
please check working well.

**Comment by zenkhas** [comment]
![screenshot_1521543066](https://user-images.githubusercontent.com/15236686/37650359-87e1cdac-2c56-11e8-9a60-eb81f356c1ee.png)
@skydoves Thank you for your response buddy. Unfortunately it didn't fixed it either. Please find the screenshot above.

**Comment by skydoves** [comment]
I think the only way now to fix is using setHeight() method.

```java
    private OnItemListener onItemListener = new OnItemListener() {
        @Override
        public void onClick(View view, String name) {
            hamburgerMenu.setHeight(600);
            hamburgerMenu.showAsDropDown(view, 0, 0);
        }
    };
```

It looks a bug and works differently by devices.
In my case, Samsung Galaxy s5, s6, and s7 work well (auto-scrolling or positioning).
but in s4 works strange and android studio's virtual machine is the same.
because it's height is not fixed.
if I find other alternatives, I will get in touch with you. Thanks!

**Comment by skydoves** [comment]
@zenkhas sorry for late.
it fixed at v2.0.3.
but it looks more nature using with

```java
.setAnimation(MenuAnimation.FADE)
```
